.fixtures.yml

```diff
@@ -1,3 +1,11 @@
 fixtures:
   repositories:
-    stdlib: https://github.com/puppetlabs/puppetlabs-stdlib
+    facts: 'https://github.com/puppetlabs/puppetlabs-facts.git'
+    puppet_agent:
+      repo: 'https://github.com/puppetlabs/puppetlabs-puppet_agent.git'
+      ref: v4.13.0
```

Comment:
Why is that pinned?

Reply:
We are using same version across all the modules, so kept the convention here as well, as its pulling from github.